openapi/p4pa-pu-bff.openapi.yaml

```diff
@@ -2542,3 +2542,41 @@ components:
         - ALL
         - NONE
         - SELECTED
+    PaidExportFileRequestDTO:
+      type: object
+      required:
+        - organizationId
+        - exportFileType
+        - fileVersion
+        - paymentDate
+        - debtPositionTypeOrgId
+      properties:
+        organizationId:
+          type: integer
+          format: int64
+        exportFileType:
+          type: 'ExportFileType'
+        fileVersion:
+          type: string
+        paymentDate:
+          type: 'LocalDateInterval'
+        debtPositionTypeOrgId:
+          type: integer
+          format: int64
+    ReceiptsArchivingExportFileRequestDTO:
```

Comment:
This structure is misaligned with BE model, is this wanted?
BE group filter fields in a nested object, while here you are flattening this structure.
There is a reason for this?

Classification Export will have many fields... this could be have some negative effect on read the common structure...